src/main/java/com/questhelper/steps/QuestStep.java

```diff
@@ -636,8 +637,7 @@ protected void renderInventory(Graphics2D graphics, WorldPoint worldPoint, List<
 				if (distanceLimit)
 				{
 					WorldPoint playerLocation = client.getLocalPlayer().getWorldLocation();
-					WorldPoint goalWp = QuestPerspective.getWorldPointConsideringWorldView(client, client.getTopLevelWorldView(), worldPoint);
-					if (goalWp != null && playerLocation.distanceTo(goalWp) <= 100) continue;
+					if (definedPoint.distanceTo(playerLocation) <= 100) continue;
```

Comment:
It looks like `QuestPerspective.getWorldPointConsideringWorldView` used to handle `worldPoint` being null, but we don't do a `definedPoint` null check here.
The null check should probably be added back, unless you've ensured the paths to this function already have a check, in which case we should mark the `DefinedPoint` as `@Nonnull`

Reply:
The reason I'm not saying we 100% need it is because of the "unless you've ensured the paths to this function already have a check, in which case we should mark the `DefinedPoint` as `@Nonnull`" statement